docs-ref-conceptual/create-an-azure-service-principal-azure-cli.md

```diff
@@ -173,25 +199,21 @@ The Azure CLI has the following commands to manage role assignments:
 * [az role assignment create](/cli/azure/role/assignment#az-role-assignment-create)
 * [az role assignment delete](/cli/azure/role/assignment#az-role-assignment-delete)
 
-We recommend you use the **Contributor** role at minimum for a service principal. This role has full permissions to read and write to an Azure account. The **Reader** role is more restrictive, with read-only access. For more information on Role-Based Access Control (RBAC) and roles, see [RBAC: Built-in roles](/azure/active-directory/role-based-access-built-in-roles).
+The **Contributor** role has full permissions to read and write to an Azure account. The **Reader** role is more restrictive, with read-only access. For more information on Role-Based Access Control (RBAC) and roles, see [RBAC: Built-in roles](/azure/active-directory/role-based-access-built-in-roles).
 
-This example adds the **Reader** role and removes the **Contributor** one:
+This example adds the **Reader** role and removes the **Contributor** role:
 
 ```azurecli-interactive
-az role assignment create --assignee APP_ID --role Reader
-az role assignment delete --assignee APP_ID --role Contributor
+az role assignment create --assignee appID --role Reader
+az role assignment delete --assignee appID --role Contributor
 ```
 
-> [!NOTE]
-> If your account doesn't have permission to assign a role, you see an error message that your account "does not have authorization to
-> perform action 'Microsoft.Authorization/roleAssignments/write'." Contact your Azure Active Directory admin to manage roles.
-
-Adding a role _doesn't_ restrict previously assigned permissions. When restricting a service principal's permissions, the __Contributor__ role should be removed.
+Adding a role _doesn't_ restrict previously assigned permissions. When restricting a service principal's permissions, the __Contributor__ role should be removed if previously assigned.
 
 The changes can be verified by listing the assigned roles:
 
 ```azurecli-interactive
-az role assignment list --assignee APP_ID
+az role assignment list --assignee appID
```

Comment:
I think we need better annotation to mark it as a placeholder, such as `{appId}`.

Reply:
https://github.com/Azure-Samples/azure-cli-samples/blob/master/sql-database/backup-database/backup-database.sh puts placeholders in a variable, like `$server`. However, `appID` looks like a const. 

Personally I like the Python placeholder convention - `{appId}`.

**Indeed, we need a finalized convention for placeholders.**